scripts/update-baseline.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+if [[ "$(pwd)" != *mapbox-directions-swift ]]; then
```

Comment:
Should we go with `SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"` as it's not required to have this repo in `mapbox-directions-swift` dir.